core/src/main/java/org/axonframework/serialization/compression/AbstractSerializerCompressionWrapper.java

```diff
@@ -0,0 +1,117 @@
+package org.axonframework.serialization.compression;
+
+import org.axonframework.serialization.*;
+
+/**
+ * Wrapper around an embedded {@link Serializer} that uses a byte[] as a working format
+ * to compress the {@link SerializedObject} from the embedded serializer. Implementations must implement the
+ * {@code doCompress} and {@code doDecompress} methods.
+ *
+ * @author Michael Willemse
+ */
+public abstract class AbstractSerializerCompressionWrapper implements Serializer {
+
+    private final Serializer embeddedSerializer;
+    private final Converter converter;
+
+    /**
+     * Initializes AbstractByteArrayCompressionSerializer implementation with the embedded serializer as parameter
```

Comment:
You're stating to initialize a `AbstractByteArrayCompressionSerializer` whilst this is a `AbstractSerializerCompressionWrapper`. Is that correct?